freeipa/src/main/java/com/sequenceiq/freeipa/controller/validation/CreateFreeIpaRequestValidator.java

```diff
@@ -27,9 +27,27 @@
 
     @Override
     public ValidationResult validate(CreateFreeIpaRequest subject) {
+        // FreeIPA's recommends no more than 4 replication agreements per node. To simply things, only allow a fully connected mesh.
+        // Also don't allow a special case with 5 nodes for 1 group when the there is a maximum of 4 instances when using multiple groups.
+        final int maxInstances = 4;
+
+        final int maxInstanceGroups = 1;
```

Comment:
I think these should be class level constants, or variables configurable via env parameters, so we can override them